crawler/cmd/download-whitelist.go

```diff
@@ -62,18 +64,20 @@ var downloadWhitelistCmd = &cobra.Command{
 	REPOLIST:
 		for _, i := range repolist.Registrati {
 			for idx, publisher := range publishers {
-				if publisher.CodiceIPA == i.IPA {
-					// If this IPA code is already known, append this URL to the existing item
-					publishers[idx].Organizations = funk.UniqString(append(publisher.Organizations, i.URL))
+				if publisher.Id == i.IPA {
+					u, _ := url.Parse(i.URL)
```

Comment:
[**warning**] The avoid of logging or returning error in URL Parsing is wanted? (also line 75)

Reply:
This was a first pass of refactoring, with the aim of making it use `url.URL` where possibile and make the upcoming patches stand on an improvement.

In this particular case, I'd like to get rid of the `download-whitelist` command altogether (it makes a `GET` and transform the results to a different structure, but I think we can just refactor the original API and make that unnecessary - the crawler is the only consumer of that endpoint) so I won't worry about the URL being valid (which it should, as we get it from https://github.com/italia/developers-italia-onboarding that validates the URLs).

Besides, we realized with @sebbalex a while ago that `url.Parse()` is reaaaally hard to make error out, rendering it unsuitable to make useful URL validation: https://go.dev/play/p/fLMZtU3bcpE. We should have an issue somewhere that mentions better URL validation on our go projects.